src/sentry/static/sentry/app/views/performance/trends/changedTransactions.tsx

```diff
@@ -296,7 +333,20 @@ function TrendsListItem(props: TrendsListItemProps) {
       </ItemRadioContainer>
       <ItemTransactionNameContainer>
         <ItemTransactionName>
-          <TransactionLink {...props} />
+          <Tooltip
+            title={
+              <TooltipContent>
+                <span>{t('Total Events')}</span>
+                <span>
+                  <Count value={transaction.count_range_1} />
+                  {' → '}
```

Comment:
Should this be an arrow icon?

Reply:
Yeah it should, I'll fix it later though.